node/test/dialog.spec.ts

```diff
@@ -1072,7 +1092,68 @@ describe('dialog', async () => {
       ]);
     });
 
-    it('responds with a random neutral feedback message when the confidence is below 90% and the lesson is sensitive', async () => {
+    it('responds with a random sensitive positive feedback message when lesson is sensitive', async () => {
+      if (mockAxios) {
```

Comment:
probably also a merge w emily changes, but let's get rid of the `if(mockAxios)`...we no longer support unmocked running of these tests and don't want to add new instances suggesting it